packages/zowe-explorer/__tests__/__unit__/dataset/DatasetTree.unit.test.ts

```diff
@@ -497,6 +497,7 @@ describe("Dataset Tree Unit Tests - Function loadProfilesForFavorites", () => {
         expect(resultFavProfileNode).toEqual(expectedFavProfileNode);
     });
     it("Checking that error is handled if profile not successfully loaded for profile grouping node in Favorites", async () => {
+        jest.restoreAllMocks();
```

Comment:
Since we are just concerned with clearing the old data from `showErrorMessageSpy` here, a less heavy-handed way to do this might be to just use [Jest's `mock.mockClear()` function](https://jestjs.io/docs/mock-function-api#mockfnmockclear) immediately after the previous time the spy was used. You can try adding `showErrorMessageSpy.mockClear();` after the `expect` statement on [line 254](https://github.com/zowe/vscode-extension-for-zowe/blob/master/packages/zowe-explorer/__tests__/__unit__/dataset/DatasetTree.unit.test.ts#L254), and removing the `restoreAllMocks` at line 500.